evap/evaluation/templates/evap_course_edit_js.html

```diff
@@ -67,8 +59,7 @@
             return id || (name && name != "---------");
         };
         row_added = function(row) {
-            twipsify();
-            assignClickHandlers()
+            assignClickHandlers();
```

Comment:
please elaborate. i was talking about the rows that are added via javascript, because the code in the base file is run only once.